src/lib/server/session.ts

```diff
@@ -16,7 +16,9 @@ export async function deleteSession(id: string) {
 }
 
 export async function createSession(user: UsersRecord) {
+	// how this works?
+	// it seems to be linking the user/user.id to the users collection/table
```

Comment:
It creates a session entry in the DB that links to that user's entry in the Users table, and the session entry's ID is stored as the 'sessionid' cookie.  Whenever the user tries to access the app, if the sessionid cookie exists, then the FB login step is skipped and this entry is used to determine who the user is; otherwise the user has to login through FB to create a new session/cookie.